Sources/ArcGISToolkit/Components/FeatureFormView/Subviews/FormElements/UtilityAssociationsFormElementView.swift

```diff
@@ -60,23 +60,20 @@ private struct UtilityAssociationGroupResultView: View {
     
     var body: some View {
         List(utilityAssociationGroupResult.associationResults, id: \.associatedFeature.globalID) { utilityAssociationResult in
-            UtilityAssociationResultView(
-                selectionAction: {
-                    let navigationAction: () -> Void = {
-                        navigationLayerModel.push {
-                            InternalFeatureFormView(
-                                featureForm: FeatureForm(feature: utilityAssociationResult.associatedFeature)
-                            )
-                        }
-                    }
-                    if internalFeatureFormViewModel.featureForm.hasEdits {
-                        setAlertContinuation?(true, navigationAction)
-                    } else {
-                        navigationAction()
+            UtilityAssociationResultButton(utilityAssociationResult) {
+                let navigationAction: () -> Void = {
```

Comment:
No need to declare the type here.
```suggestion
                let navigationAction = {
```